jaxmarl/environments/jaxnav/maps/grid_map.py

```diff
@@ -330,6 +342,75 @@ def _passable(grid, posa, posb):
         
     def scale_coords(self, x):
         return x / self.cell_size
+    
+    def dikstra_path(self, map_data, pos1, pos2):
```

Comment:
Did you implement dijkstra's algorithm in Jax 😆 

absolutely nuts good work

Reply:
Would be good to describe how this works in a high level comment

Reply:
at 11pm on a sunday yes haahah, Reviewer 4 asked for environment metrics..!